### PR TITLE
Add optional argument to allow setting pull secret file

### PR DESCRIPTION
**Pull request by skorobogatydmitry**

### CODE REVIEW

File: stable/yugaware/values.yaml [hunk 1]
```diff
@@ -11,7 +10,10 @@ image:
   repository: quay.io/yugabyte/yugaware
   tag: 2.5.1.0-b153
   pullPolicy: IfNotPresent
-  pullSecret: yugabyte-k8s-pull-secret
+  pullSecret: yugabyte-k8s-pull-secret-1
```
Comment (bhavin192) [inline stable/yugaware/values.yaml]: We should not change the default name here. If someone has existing deployment, and they upgrade, image pull will fail in this case.

Comment (skorobogatydmitry) [reply stable/yugaware/values.yaml]: Good catch!

**Comment by skorobogatydmitry** [comment]
Any other comments?
@bhavin192 @iSignal @rtsisyk

File: stable/yugaware/values.yaml [hunk 2]
```diff
@@ -12,6 +11,9 @@ image:
   tag: 2.5.1.0-b153
   pullPolicy: IfNotPresent
   pullSecret: yugabyte-k8s-pull-secret
+  # Docker config JSON File name
+  # If set, this file content will be used to automatically create secred named as above
```
Comment (iSignal) [inline stable/yugaware/values.yaml]: s/secred/secret

**Review by iSignal — APPROVED** [review]
Looks good to me. @bhavin192 : please request changes if you have any other comments?

File: stable/yugaware/values.yaml [hunk 3]
```diff
@@ -12,6 +11,9 @@ image:
...
+  # pullSecretFile:
```
Comment (bhavin192) [inline stable/yugaware/values.yaml]:
```suggestion
  ## Docker config JSON File name
  ## If set, this file content will be used to automatically create secret named as above
  # pullSecretFile:
```

Just the convention of having comments with double hash and actual YAML in single hash.

**Review by bhavin192 — APPROVED** [review]
Only a minor formatting change. Else looks good.

**Pull request merged**